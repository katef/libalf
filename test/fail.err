error: Missing escape
at 3: 'abc\'
----------^
error: Unrecognised escape
at 3: 'abc\x'
----------^^
error: Status overflow
at 4: 'abc%0'
-----------^
error: Status overflow
at 4: 'abc%000'
-----------^^^
error: Missing directive
at 3: 'abc%65529'
----------^^^^^^
error: Status overflow
at 4: 'abc%65530'
-----------^^^^^
error: Too many statuses
at 261: 'abc%1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,4,4,4,4,4,4,4,4,4,4,5,5,5,5,5,5,5,5,5,5,6,6,6,6,6,6,6,6,6,6,7,7,7,7,7,7,7,7,7,7,8,8,8,8,8,8,8,8,8,8,9,9,9,9,9,9,9,9,9,9,1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,4,4,4,4,4,4,4,4,9'
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------^
error: Missing closing brace
at 4: 'abc%{xyz'
-----------^
error: Empty name
at 4: 'abc%{}xyz'
-----------^^
error: Unrecognised directive
at 3: 'abc%{xyz}-x'
----------^^^^^^^
error: Unrecognised directive
at 3: 'abc%>{xyz}-t'
----------^^^^^^^^
error: Too many redirect flags
at 4: 'abc%<>>>---'
-----------^^^^
error: Too many redirect flags
at 4: 'abc%<>>>{xyz}t'
-----------^^^^
error: Missing directive
at 3: 'abc%<'
----------^^
error: Unrecognised directive
at 3: 'abc%<_xyz'
----------^^^
error: Unrecognised directive
at 3: 'abc%<{xyz}-xyz'
----------^^^^^^^^
error: Unrecognised directive
at 3: 'abc%<{xyz}^xyz'
----------^^^^^^^^^
error: Missing name
at 3: 'abc%C'
----------^^
error: Missing name
at 3: 'abc%<C'
----------^^^
error: Name overflow
at 6: 'abc%<{0000000000111111111122222222223333333333444444444455555555556666666666777777777788888888889999999999000000000011111111112222222x}C'
-------------^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
error: Unwanted name
at 4: 'abc%{xyz}z'
-----------^^^^^
error: Unrecognised ip type
at 5: 'abc%{xyz}a'
------------^^^
error: Unrecognised port type
at 5: 'abc%{xyz}p'
------------^^^
error: Unrecognised id type
at 5: 'abc%{xyz}P'
------------^^^
error: Unrecognised rtime unit
at 5: 'abc%{xyz}T'
------------^^^
error: Unrecognised directive
at 3: 'abc%{xyz}^x'
----------^^^^^^^^
error: Unrecognised directive
at 3: 'abc%{xyz}^tx'
----------^^^^^^^^^
error: Missing directive
at 3: 'abc%'
----------^
error: Missing directive
at 3: 'abc%{xyz}'
----------^^^^^^
error: Unrecognised directive
at 3: 'abc%_'
----------^^
error: Unrecognised directive
at 3: 'abc%{xyz}_'
----------^^^^^^^
