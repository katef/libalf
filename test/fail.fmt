abc\
abc\x
abc%0
abc%000
abc%65529
abc%65530
abc%1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,4,4,4,4,4,4,4,4,4,4,5,5,5,5,5,5,5,5,5,5,6,6,6,6,6,6,6,6,6,6,7,7,7,7,7,7,7,7,7,7,8,8,8,8,8,8,8,8,8,8,9,9,9,9,9,9,9,9,9,9,1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,4,4,4,4,4,4,4,4,9
abc%{xyz
abc%{}xyz
abc%{xyz}-x
abc%{xyz}t
abc%>{xyz}-t
abc%<>>>---
abc%<>>>{xyz}t
abc%<
abc%<_xyz
abc%<{xyz}-xyz
abc%<{xyz}^xyz
abc%C
abc%<C
abc%<{0000000000111111111122222222223333333333444444444455555555556666666666777777777788888888889999999999000000000011111111112222222x}C
abc%{xyz}z
abc%{xyz}a
abc%{xyz}p
abc%{xyz}P
abc%{xyz}T
abc%{xyz}^x
abc%{xyz}^tx
abc%
abc%{xyz}
abc%_
abc%{xyz}_
