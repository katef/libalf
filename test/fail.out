literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
>strftime: (when=0, fmt=xyz)
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
literal: 'a'
literal: 'b'
literal: 'c'
